system.config.js

```diff
@@ -37,10 +38,14 @@ System.config({
     "lodash": "npm:lodash@4.15.0",
     "plugin-babel": "npm:systemjs-plugin-babel@0.0.13",
     "plugin-babel-runtime": "npm:babel-runtime@5.8.38",
+    "querystring": "github:jspm/nodelibs-querystring@0.1.0",
```

Comment:
jspm should have deleted this... I think running `jspm clean` should get rid of them if they are no longer in package.json